### PR TITLE
updated to angular v16

### PR DESCRIPTION
**Pull request by onlyargon**
updated library to angular 16

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ng-image-slider",
-  "version": "7.0.1",
+  "name": "ng-image-slider-v16",
```
[the remark this answers is not in the document]
Comment (onlyargon) [reply package.json:2]: let's do that..

[the remark this answers is not in the document]
Comment (sanjayV) [reply package.json]: @onlyargon 
we need to update it in all files where you mentioned **-v16**

[the remark this answers is not in the document]
Comment (onlyargon) [reply package.json]: I have updated that too. please review

**Comment by KAAC-78** [comment]
Hi! Are you planning to complete this PR? @sanjayV  @onlyargon

**Comment by onlyargon** [comment]
> Hi! Are you planning to complete this PR? @sanjayV @onlyargon

Changes have been made but PR hasn't been reviewed or approved. If you are in hurry use this https://www.npmjs.com/package/ng-image-slider-v16

**Comment by coderpradp** [comment]
Hope this gets merged soon

**Comment by sanjayV** [comment]
[Package Version 8.0.0](https://www.npmjs.com/package/ng-image-slider/v/8.0.0) released for Angular version 16+
https://www.npmjs.com/package/ng-image-slider/v/8.0.0

**Pull request closed without merging**